modules/ROOT/pages/provisioning-vmware.adoc

```diff
@@ -92,9 +92,40 @@ ssh core@<ip address>
 
 This section shows how to use vSphere facilities to configure and run VMs from the command-line. Similar steps can be performed via the graphical UI too.
 
-=== Importing the OVA
+TIP: While the examples below use `govc session.login` to authenticate, you can also use environment variables to provide credentials. Check the https://github.com/vmware/govmomi/tree/main/govc#usage[official documentation] for details.
```

Comment:
URL could also be set to `https://github.com/vmware/govmomi/blob/main/govc/README.md#usage` … not sure whether it makes much of a difference?